packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -56,6 +62,7 @@ export const navListClass = style([
   }),
   {
     listStyle: 'none',
+    paddingInlineStart: '1rem',
```

Comment:
this can be updated to use atoms

Reply:
this will be included in another PR where we'll adapt to the use of atoms in the pages